08-Placing-Barriers/03-Wall-geometries.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
<p>small typo in the variable name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/#comment-1392966268'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Fixed.